src/content/en/2021/capabilities.md

```diff
@@ -1,20 +1,368 @@
+
+1. The number one site is <a hreflang="en" href="https://whatwebcando.today/">whatwebcando.today</a>, which uses 28 capabilities. It showcases different HTML5 device integration APIs by providing a live demo for every capability. Naturally, the number of used APIs is very high. In the result set, a similar site called <a hreflang="en" href="https://whatpwacando.today/">whatpwacando.today</a> showcases PWA capabilities and uses eight APIs.
+1. The <a hreflang="en" href="https://polisnotis.se/">PolisNotis</a> PWA shows police notices in Sweden. It uses ten APIs, including the Declarative Link Capturing API to define that the PWA should always open a new window when clicking a PWA-related link. The Web Share API is used in the source code, but the sharing functionality is not exposed to the UI. The app also uses the Badging API to alert the user via the app icon if there is a new notice.
+1. The website <a hreflang="en" href="https://system-scanner.net/">System Scanner</a> uses nine APIs: It shows an overview of the system information exposed by the browser, including sensor information provided by the Generic Sensor API.
+1. Eight sites use eight Fugu APIs: One of them is the aforementioned <a hreflang="en" href="https://excalidraw.com/">Excalidraw</a>, an online drawing tool for creating drawings in a hand-drawn style. As a traditional productivity app, it benefits from the new capabilities.
+
+Some websites from the result set are Internet forums based on <a hreflang="en" href="https://www.discourse.org/">Discourse</a>. This forum software supports a total of eight Fugu APIs. Discourse-based forums are installable and support, among others, the Badging API to show the amount of unread notifications.
+
+The results also include sites that aren't proactively using the APIs. For example, some sites ship library code that could theoretically access the capabilities. Some sites check for the presence of Fugu APIs to determine the user's browser.
+
+## Conclusion
+Capabilities help move the web forward by unlocking more and more use cases for developers. As this chapter shows, developers use the new web platform APIs to build powerful applications. In contrast to their platform-specific counterparts, those applications don't necessarily need to be installed to the system and don't require any additional third-party runtimes or plug-ins to work. They run on any platform that can run a powerful browser.
+
+One example of this concept working is Visual Studio Code. This application has always been web-based, but it still relied on platform-specific application wrappers like Electron. Thanks to capabilities like the File System Access API, Microsoft was able to release the application as a browser application (<a hreflang="en" href="https://vscode.dev">vscode.dev</a>) in October 2021. Almost all features work here, except debugging or terminal access, since there is no capability for this yet.
```

Comment:
Maybe hint at the future?

```suggestion
One example of this concept working is Visual Studio Code. This application has always been web-based, but it still relied on platform-specific application wrappers like Electron. Thanks to capabilities like the File System Access API, Microsoft was able to release the application as a browser application (<a hreflang="en" href="https://vscode.dev">vscode.dev</a>) in October 2021. Almost all features work here, except debugging or terminal access, since there is no capability for this (yet!).
```

Reply:
I don't expect these APIs to land in the near future, but yes, we can go with that.